### PR TITLE
[MAIN] move QrDecryptor to crypto lib

### PR DESCRIPTION
**Pull request by AmrMohammed89**

### CODE REVIEW

**Comment by MarjanaKarzek** [comment]
Here my personal opinion on this: the QR code and related code and decryption flow are our business logic, it should be part of our domain layer and not extracted to krypt. If there is code that can be abstracted enough to serve as generic crypto code like general SHA-256 hashing or AES en/decryption, then the right location is the krypt library. So to summaries: these files should be in the Vivy app code base, not in the krypt library.

**Comment by sulemanshakil** [comment]
@MarjanaKarzek i suggested @AmrMohammed89 to move this code in krypt to unblock him because androidTest are not working in android code base. here we can write test.

**Review by sulemanshakil — APPROVED** [review]

**Comment by MarjanaKarzek** [comment]
What does that mean, that the Android tests are not working?

File: vivy_encryptor/src/androidTest/java/com/vivy/qr/decryption/QrDecryptorTest.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
...OrdqcikVKUEv0fGlcT67DH6CyoKyA8Pd0E+/zyJUzkoWhCBJig4ljfR+sxDOSvk8CPbU2gzyMC325BFXZ3lnyYwFtesMRBGS3prOvB92raeanOQruiRoCh9ZfZFoSlDFTXHLF/D/tIXMFwfWYvr3NwWlEQX46g1OjNwHNS19760+y8yowtcsvpQXH3ifL6aTGpud8nuqGxiCiITXsliIJY/Rco4S8fn63mWfsR2JB69/vexxoMZTZgLe8tWQHskVgRIUdWjliVDAHwv6Q8HASU3cZPh1i2CGJaJayTl8qXGu+mYGXYuSz0r5I6f8IBKun2xRgrH3bt60MaRmCl4WWO6v1bs2/YFppNu07AFlraJz1HtBwfN/aVkbU8Cd7hbzfD4W6v6qqo0pGV3JplLFTgScusQld3QyDqDNyAc3FAYvYANywIddeACggu/79+WMzjll535NgE08r34SPJL8ipPLTDKj4hFupRWHYfVjIMZ57S6ZECVFaBOy8JgUjL2qLQj4wYLUASJBvgcsucPMnVVD9AIp4kdDuJ4Z/8mxdBHwPhwwpRYK8iTjkj4r9ayXr2Vsa8cB8kUhcgyldFYdRcvYI/RUelyvCunfWPBSIU86wyEeSRGiNcW7/4yAQxxQgAcUGOL/xKnZracaLgBVIJwwQ8FCXfeTmzPfhJ52C/NA6sqCufEIBRV8tMnl9KlWCt/rrEnbD1xqLH4A9mFtOqNW5D3vHYSeI0a842JR5gBo1/O2UiEmaGGDTWHnRHQtaDEKaZzXg/rMDZw6H3LHxLnlzjPjlL5RpFFb0+ardroNBtu8lbqIxRZez6MyleHPrGwMQ8rxH/ANFmm4zlZAXfFnWAnaE77mvojBRMRFN4nCf3Kp3wWtb74ecaUE8l9eXbGqTqrmeehuMHGmBMyTMrcA1HLR6Prb6QWmooPntHNYA/zB+/BNYcP9BfHosVXSphj/sZiYUPULaqEAltzbfv9Ot5lcbQDQYL92ZcYyO/0SlxZZ69YtPzcEisi9Z5WT4Q4eId7DoKOyf1amthyWddabo2DDflaR7Di0ixMBqmM43n6P9R96ixW/HIySWT6/3FlYJBp53TNH3Z9Ld2eWreS5wht6SQqPB0Ce51Uwm7Pvtdo3tukgfebB3CEBKI8Cwgc9IonDpmf9t2q4QFhDeAGLrkYZZmsRFHykwgMch+EFw=="
```
Comment (sulemanshakil) [inline vivy_encryptor/src/androidTest/java/com/vivy/qr/decryption/QrDecryptorTest.kt:16]: please move them inside some constant file

**Comment by AmrMohammed89** [comment]
> What does that mean, that the Android tests are not working?

androidTest isn't working in legacy module, needs to invest sometime to fix it

**Pull request merged**